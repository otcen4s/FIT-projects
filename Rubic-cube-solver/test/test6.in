452
111
111
333 441 543 622
222 331 543 655
221 543 644 555
664
663
662