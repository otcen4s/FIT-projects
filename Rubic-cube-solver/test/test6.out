452
111
111
333 441 543 622
222 331 543 655
221 543 644 555
664
663
662

452
111
111
333 441 543 622
222 331 543 655
543 644 555 221
432
666
666

452
111
111
333 441 543 622
222 331 543 655
644 555 221 543
266
366
466

452
111
111
333 441 543 622
222 331 543 655
555 221 543 644
666
666
234

111
111
111
333 444 555 222
222 333 444 555
555 222 333 444
666
666
666

111
111
111
222 333 444 555
222 333 444 555
555 222 333 444
666
666
666

111
111
111
222 333 444 555
222 333 444 555
222 333 444 555
666
666
666

